script/Ciel/Day 13/13_01_02_CIEL13_5_2.txt

```diff
@@ -311,42 +345,55 @@ I didn't even stop to think about what that was like for her.
 [sha:4c80cb92df3170848f0e3c1e81b2c8fc33cae67e]{
 -- Page 87, Offset 35506.
 -- “……昔、私はこの体になるきっかけで罪を犯しました”
-"...A long time ago, I committed a sin when I was granted this body."
+// Referencing 28018
+// Voiced
+"Back when I first came into possession of this body... I committed a sin."
 }
 [sha:188f583daa5e8c906adb8757a8d3fdc515973e21]{
 -- Page 87, Offset 35507.
 -- “多くの間違いをおかして、多くの人命を見殺しにした”
-"I made numerous mistakes, and watched as countless people died right before my eyes."
+// Referencing 28019
+// Voiced
+"I made a great many mistakes,
+ and so caused a great many people to die."
 }
 [sha:72e5dd355d2d71c453988acc9d1ee84a5386b8dd]{
 -- Page 87, Offset 35508.
 -- “その後、私に課せられたのは犠牲になった人々の数と同じだけの処罰でした”
-"After that, I was sentenced to a punishment equivalent to the number of victims who were sacrificed."
+// Referencing 28020
+// Voiced
+"Afterward, I was sentenced to a punishment commensurate with the number of victims."
 }
 [sha:40dca8e7edfda57e8c13e9c58e0cf0fd7d206c88]{
 -- Page 88, Offset 35509.
 -- ふざけてる。なんの悪夢だそれは。
-You're kidding me. What kind of nightmare is that?
+What a sick fucking joke.
+ That's worse than a nightmare.
 }
 [sha:b87c60042bc367c4fb9f3d5b41d61777d6811274]{
 -- Page 88, Offset 35510.
 -- 今の俺と同じ、自分の意思が残ったままロアの行為を見続けて、ロアとしてアルクェイドに殺された。
-Similarly to me, even though her will still remained intact, she was forced to watch as Roa carried out his plan until Arcueid killed her.
+Just like me, her will remained intact.
+ She was forced to watch all the atrocities Roa committed,
+ and was killed alongside him by Arcueid.
```

Comment:
I think this should still be one big sentences. The connection in the first clause feels a bit too diluted.
>...remained intact as she was forced to watch the atrocities Roa committed, all the way up until she was killed alongside him by Arcueid.